### ISSUE TITLE
Retrocompatibility of the library

### ISSUE THREAD

**Issue by lionelkusch** [mind-inria/hidimstat#83]
Once the tests for the project are validate. We need to identify what are the oldest version of the packages valid for running the packages.

**Comment by lionelkusch** [comment]
The compatibility with Scikit-learn is a good starting points.

**Comment by lionelkusch** [comment]
The PR #176 updated the dependency for the library and the tests but the dependencies for the documentation and example need to be updated later.

**Comment by bthirion** [comment]
Dependecies updates should alweys be sunchronized between code, documentation and examples.

**Comment by lionelkusch** [comment]
Yes, I want to say that the dependencies for the generation of the documentation and the examples weren't yet refactored.

**Comment by lionelkusch** [comment]
This was done in #321
